src/components/accounts/CreateAccount.js

```diff
@@ -67,6 +66,18 @@ class CreateAccount extends Component {
         token: ''
     }
 
+    componentDidMount = () => {
+        const { loginError, resetAccounts } = this.props;
```

Comment:
what is this change trying to do?

Reply:
this component receives the url params for linkdrops and sets the user's desired accountId to a tempAccount we use internally before they complete recovery setup

Reply:
I'm trying to understand `componentDidMount` specifically does, not whole component. E.g. when `resetAccounts` can be called, etc

Reply:
@Patrick1904 ?

Reply:
I'm not sure why. Removed it.